### DISCUSSION TITLE
UDFs requiring external_access_integrations - how to specify?

### DISCUSSION THREAD

**Discussion by cwhatley** [Snowflake-Labs/terraform-provider-snowflake#3146] [Q&A] [answered]
I have functions I'd like to manage with terraform that require an external_access_integrations parameter. Is there some way to do that in the current implementation?

**Comment by sfc-gh-asawicki** [comment, marked as answer]
Hey @cwhatley. Thanks for reaching out to us.

Functions are currently being reworked and we will add the missing attributes to the resources (resources because we are splitting the existing snowflake_function into dedicated resources - for each type).

For now, you can use [snowflake_unsafe_execute](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/unsafe_execute).

**Reply by cwhatley** [reply]
Very relieved to have a workaround. Thanks!
